OpenOversight/tests/test_models.py

```diff
@@ -173,8 +173,7 @@ def test_salary_repr(mockdata):
 
 def test_password_not_printed(mockdata):
     user = User(password="bacon")
-    with raises(AttributeError):
-        user.password
```

Comment:
Not entirely sure what this was testing.

Reply:
This is testing that you can set but not recover the plaintext password through user.password
https://github.com/lucyparsons/OpenOversight/blob/5e0e0d711e287e15e664c99dde1fd364443dd247/OpenOversight/app/models/database.py#L752-L761

Reply:
I can change the assert in that case!

Reply:
Done!

Reply:
I modified it because it just left the hanging `user.password` statement, but I could modify it so that it validates both conditions.

Reply:
Done done.